lib/content-services/document-list/components/document-list.component.ts

```diff
@@ -618,34 +616,12 @@ export class DocumentListComponent implements OnInit, OnChanges, OnDestroy, Afte
             this.setupDefaultColumns(this._currentFolderId);
         }
 
-        this.loadFolderByNodeId(this._currentFolderId);
-    }
-
-    loadFolderByNodeId(nodeId: string) {
-        if (this.customResourcesService.isCustomSource(nodeId)) {
-            this.updateCustomSourceData(nodeId);
```

Comment:
updateCustomSourceData is not present in your PR why?

Reply:
fixed